DTS-STN/Service-Canada-Labs
# OAS Page Design Updates + Blog Page

# OAS Page Design Updates + Blog Page

This PR includes several things:

1. Updates to the OAS overview page design
2. Added new dynamic route `[id].js` under `/pages/projects/oas-benefits-estimator/[id].js` to support the blog posts/updates
3. Hooked up AEM data to new `[id].js` (will want to move to markdown for the majority of the content on these kinds of pages and work on building the client frontend programatically)
4. Updated OAS homepage card to include tag indicating there is a new update (currently hardcoded for OAS, will look into a programatic implementation later)

## Test Instructions

1. Navigate to home page
5. Go to OAS overview
6. See that design matches spec [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=14952-40411&mode=design&t=SKnL4LWuybZYkqCZ-4)
7. Go to the update through the card
8. See that update page design matches spec [here](https://www.figma.com/file/ke3SrZNGMP3AjwzMMuEgbo/%5Bcurrent%5D-BDM-DECD-SC-Labs?type=design&node-id=14948-40097&mode=design&t=SKnL4LWuybZYkqCZ-4)
9. Repeat this for both languages, switch languages mid-navigation

## Definition of Done
